bin/s3_store_server

```diff
@@ -2,6 +2,8 @@
 # frozen_string_literal: true
 if ARGV[1].nil?
   system("ruby", "bin/app.rb", ARGV[0])
-else
+elsif ARGV[2].nil?
   system("ruby", "bin/app.rb", ARGV[0], ARGV[1])
+else
+  system("ruby", "bin/app.rb", ARGV[0], ARGV[1], ARGV[2])
 end
```

Comment:
Is this `if-elsif-else` block required?

Reply:
I was wondering the same thing, but way checks the arguments strictly, and we cannot specify hash or array in here. So have for different number of arguments, we have to specify this. If you are aware of a better method to handle this let me know.

Reply:
How about `ARGV.join(' ')` ?